src/web-ui/src/public/Auth.vue

```diff
@@ -0,0 +1,166 @@
+<template>
+  <secondary-layout>
```

Comment:
Nit pick: can we use `SecondaryLayout` for casing? I know it's not the `vue` way, but it's the naming convention we've followed thus far. Perhaps something to consider changing throughout the app.